### PR TITLE
XMEM serve must use SZWEAUTH

### LINKED ISSUE #1547: XMEM server does not use SZWEAUTH
**Describe the bug**
During setup (scripts/utils/zowe-install-xmem.sh), the content of SZWEAUTH is copied to a new location, and that location is then used by the XMEM server. This implies that the server is doomed to run with outdated code, as a refresh of the Zowe libraries does not touch the copy.

**Expected behavior**
The XMEM server **must** use SZWEAUTH itself

**Additional context**

### LINKED ISSUE #1550: zowe-install-xmem.sh is looking for the wrong PROC names
**Describe the bug**
zowe-install-xmem.sh instructs zowe-copy-to-JES.sh to look for PROC names ZWESASTC and ZWESISTC, which are not used in the the ZWESASTC and ZWESISTC sample members, causing warning messages that can be easily avoided.

**To Reproduce**
invoke zowe-install-xmem.sh

**Expected behavior**
Update zowe-install-xmem.sh to check for the actual PROC names (ZWESAUX and ZWESIS01), or update the build pipeline and alter the PROC names to be ZWESASTC and ZWESISTC.

### PR DESCRIPTION
**Pull request by OnnoVdT**
Please check if your PR fulfills the following requirements. This is simply a reminder of what we are going to look for before merging your PR. If you don't know all of this information when you create this PR, don't worry. You can edit this template as you're working on it.

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Necessary documentation (if appropriate) have been added / updated
- [x] DCO signoffs have been added to all commits, including this PR

#### PR type
What type of changes does your PR introduce to Zowe? _Put an `x` in the box that applies to this PR. If you're unsure about any of them, don't hesitate to ask._ 
- [x] Bugfix 
- [ ] Feature 
- [ ] Other... Please describe:

#### Relevant issues

Fixes #1547, Fixes #1550 

#### Changes proposed in this PR

1547
- stop copying SZWEAUTH and use the real thing
- make PARMLIB optional (SZWESAMP is default)
- do not assume script is called from within scripts/utils
1550
- comment out PROC test

#### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

#### Does this PR do something the person installing Zowe should know about?

---
* Affected function: _general area of interest_ *
STC JCL
---
* Description: _1 line description_ *
update data set names in STC 
---
* Part: _name of customizable file involved_  *
SZWESAMP(ZWESASTC, ZWESISTC)

---
Before Zowe 1.14, the zowe-install-xmem.sh setup script copied
the SZWEAUTH load library and created started task JCLs
ZWESASTC and ZWESISTC that reference this copy. The related
configuration documentation instructed you to APF authorize this
copied data set.
From Zowe 1.14 on, the ZWESASTC and ZWESISTC started tasks are
expected to use load library SZWEAUTH itself, not a (possibly 
outdated) copy.

Update your active ZWESASTC and ZWESISTC PROCLIB members to use
//STEPLIB  DD DISP=SHR,DSN={zowe_hlq}.SZWEAUTH.
Also update your active PARMLIB(PROGxx) member to make 
{zowe_hlq}.SZWEAUTH APF authorized.
APF ADD DSN={zowe_hlq}.SZWEAUTH  SMS
You can dynamically mark the SZWEAUTH libary as APF authorized 
(until next IPL) using an operator command
SETPROG APF ADD,DSN={zowe_hlq}.SZWEAUTH,SMS

#### Is there a related doc issue or Pull Request? 

Doc issue/PR number: 
https://github.com/zowe/docs-site/issues/1328

### CODE REVIEW

File: scripts/utils/zowe-install-xmem.sh [hunk 1]
```diff
@@ -1,32 +1,27 @@
...
+# This program and the accompanying materials are made available
+# under the terms of the Eclipse Public License v2.0 which
+# accompanies this distribution, and is available at
 # https://www.eclipse.org/legal/epl-v20.html
 #
 # SPDX-License-Identifier: EPL-2.0
 #
-# Copyright IBM Corporation 2019, 2020
-################################################################################
+# Copyright Contributors to the Zowe Project. 2019, 2020
```
Comment (John-A-Davies) [inline scripts/utils/zowe-install-xmem.sh:10]: Steve says the IBM copyright is the right one.

Comment (OnnoVdT) [reply scripts/utils/zowe-install-xmem.sh:10]: I was told all Zowe contributions must assign copyright to "Contributors to the Zowe Project", as it's not an IBM open source project.

Comment (OnnoVdT) [reply scripts/utils/zowe-install-xmem.sh]: I did the requested update, and added the fix for #1550

**Comment by stevenhorsman** [comment]
@OnnoVdT @John-A-Davies - the tests are now failing because they are passing in a -b parameter.
- Can this parameter just be removed in the test playbook?
- In future please run tests before merging to staging to avoid breaking the staging branch

**Pull request merged**